### ISSUE TITLE
Viewing all historic versions of the schemas

### ISSUE THREAD

**Issue by Shikha-Trivedi-Saxo** [linkedin/datahub#1965]
**Is your feature request related to a problem? Please describe.**
If the schema is modified on certain point of time, the users will have different relevant versions .For example :  if the compatibility is set as forward...the producer will be dependent on V2 version schema but the consumer still uses V1 schema. So how can both producer and consumer visualize their relevant schemas on the LDH frontend

**Describe the solution you'd like**
There should be drop down on the schema aspect to visualize all historical versions of schema.

**Describe alternatives you've considered**
If all the historical data is not captured, at least last 2 versions of schemas should be available. 

**Additional context**

**Comment by maggiehays** [comment]
We are now managing feature requests through our [new portal](https://feature-requests.datahubproject.io) to improve visibility & prioritize work into our public roadmap. If this request is still relevant, please create a new issue so we can gauge current interest from the Community & prioritize work accordingly.

We'll close out this issue in the meantime. Feel free to [reach out to me on DataHub Slack](https://slack.datahubproject.io/) with any questions!
